rancher/distros-test-framework
# fix: selinux tests

#### Proposed Changes ####


- fix selinux tests that was breaking




#### Types of Changes ####



#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
- no, but will create a new one 

2. Verify code lint; we should not have errors.

 - ok fixed and the error on the lint i am fixing in the  https://github.com/rancher/distros-test-framework/pull/38 
3. Update the documentation if needed.

- n/a 
4. Update makefile and docker run if adding new tests.
- updated 

5. Run your tests at least 4 times with all configurations needed and possible.
- yes both products 

6. If needed test with different os types.
- yes, centos7 , centos8

#### Linked Issues ####



#### Further Comments ####






tests passing without upgrade and with
